homework15/js/main.js

```diff
@@ -0,0 +1,156 @@
+const API_KEY = 'uFHaNP9dBxuTVU0PU0Ek2aZ4alfaJFPi';
+const locationCode = [324561, 324505, 323903, 322162, 325343, 323037, 325523, 326609, 321985, 328328, 274663, 349727];
+let locationKey;
+
+const optionLocation = document.querySelector('.weather__location');
+const buttonGetWeather = document.querySelector('.weather__btn');
+
+function getLocationKey(){
```

Comment:
As an option, you can store cities and location codes in the object, to avoid redundant calculations
```
const CITIES_WITH_CODES = {
  Lviv: 324561,
  ...
};
```